src/snovault/elasticsearch/indexer.py

```diff
@@ -366,7 +366,7 @@ def check_log_indexing_times(self, update_infos):
             with open(self.indexer_initial_log_path, 'w', encoding='utf-8') as file_handler:
                 for update_info in update_infos:
                     str_update_info = json.dumps(update_info, ensure_ascii=False)
-                    file_handler.write(str_update_info)
+                    file_handler.write(str_update_info + '\n')
```

Comment:
Since we are on Python 3.6+, in the future, you can you f-string (https://www.geeksforgeeks.org/formatted-string-literals-f-strings-python/)

Reply:
oh yea, I forgot about this.  We should start moving toward using that in new code.